app/react-transition/connectWithStore.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I wasn't sure where to put this, so I started a `react-transition` folder. Maybe it should just be in `common/` or `common/react-transition`